common/src/java/com/zimbra/common/util/Log.java

```diff
@@ -287,6 +287,14 @@ public void warn(Object o) {
     public void warn(Object o, Throwable t) {
         getLogger().warn(o, t);
     }
+    
+    public void warnQuietly(Object o, Throwable t) {
+        if (isDebugEnabled()) {
+            getLogger().warn(o, t);
```

Comment:
Is .warn correct here? Just confirming because in above if "isDebugEnabled()" is checked.

Reply:
Fixed